### PR TITLE
Apple clone branch

### PR DESCRIPTION
**Pull request by Alaska01**
-Performed a clone of an old version of apple.com
-Create a new HTML document and a Github repository using Github guidelines
-Made a logical arrangement of all the elements and how they are grouped on the webpage.
-Laid out the basic structure of the page using empty semantic HTML elements that are appropriately sized and positioned. Made use of browser developer tools to inspect elements.
-Set up the top navigation bar. 
-Set up the main background image. 
-Fill in and position the additional sections. 
-Push your solution to Github.

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Alaska01  :wave:
-
You have done a splendid job! :tada: . Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
### [The clone](https://raw.githack.com/Alaska01/Aye-Apple-Clone/apple-clone-branch/index.html) shows good use of CSS Backgrounds and Gradients. :100: 

A few required changes: Note that for this project, we have to make sure nothing is messed up. Please see the difference in the source page and the clone below 👇 

|[Source Page](https://web.archive.org/web/20140301004610/http://www.apple.com/)|[Clone Page](https://raw.githack.com/Alaska01/Aye-Apple-Clone/apple-clone-branch/index.html)|
|:---:|:---:|
|source|clone|

- [ ] The main area of the cloned page has its contents not properly centred as with the case with the source page. Kindly fix that.
- [ ] The navbar is too long for its intended purpose. Kindly fix that.
- [ ] The four bottom contents are far spaced out. Kindly fix that.
- [ ] Remove both the horizontal and vertical scroll present in the clone.
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright).
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by Alaska01** [comment]
Thanks for the great feedback, Mr. Christian Otieno. I have made the required corrections as requested for the review. I also added a loom video of what my clone looks like and the original source clone site.

It seems the source site(https://web.archive.org/web/20140301004610/http://www.apple.com/) is loading differently on separate devices. The site is not opening for some students at all; even for me, it opens at times and, at other times, refuses to open.

Here is the link(https://www.loom.com/share/fd0162533a7f4452819f67d849304ade) to the loom video for your perusal. Thanks.

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required! :recycle: 
Hi @Alaska01, well done on this project so far. There are however some issues you need to resolve to complete this project.
- [ ] You have a `node_modules` folder in your repo. Files not related to the project should not be committed and added to .gitignore ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] Your clone appears to be broken on various points as indicated in the screenshot below.
screenshot


__Kindly make the required changes and submit another code review.__
_Feel free to reach me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

**Comment by Alaska01** [comment]
Thank you for such excellent feedback. It was quite useful in fixing the error. You may wish to use videos for feedbacks so that I may be able to see how the content displays on your system too. I have added a loom link(https://www.loom.com/share/8cba0d79affa4ff99ee57614b65ba89b) on how it is showing on my computer on various browsers. Thanks a bunch, I will be more than happy to look into any feedback you may have.

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -1,10 +1,105 @@
...
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+
+                <div class=ft-line-2>
+                    <P>Copyright © 2014 Apple Inc. All rights reserved. <span><a href="#">Terms of Use</a></span>
+                        |<span><a href="#">Privacy
+                                Policy</a></span></P>
+                </div>
```
Comment (santiago-dev96) [inline index.html:97]: You should place this box inside of the box that contains the first paragraph, it will be easier for you to style it after that.

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

## Navigation bar

Please use the same font as the page that you are trying to clone, in order to see the details of such page you should use your browser developer tools. For example, to solve this requirement I would use my developer tools to find out that the [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font) they're using is the following.

```css
font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
```

## Hero image's text

Same as the navigation bar, use a font that is similar to the one used in the original page.

## Footer

Fix the layout of the paragraphs, the first paragraph is all scrambled.

Screen Shot 2020-05-29 at 7 53 40 AM

Please submit another CRR (Code Review Request) after implementing the changes.

You can find me in slack as **srodrig**, have a nice day! :wave:

**Comment by Alaska01** [comment]
I made changes are requested by Mr. Santiago Rodriguez. The font Lucida Grande is a paid font. The reviewer suggested I use an alternative to the Lucida Grande after contacting him on slack. This was the link(https://www.quora.com/Are-there-any-free-fonts-similar-to-Lucida-Grande) I was given, and I made use of the Proza Libre font as an alternative to Lucida Grande.

The white spaces on the anchor tags were also eliminated. I hope this serves well. You may wish to view the project with the default browser setting; this is not a responsive project, and trying to view it on smaller screens or devices may lead to a distorted view. Viewing it on a browser screen with 100% view should look fine.

Thanks.

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @Alaska01,  congratulations on a great job done :+1: The project looks great and has met all the minimum requirements. You did a great job fixing all the comments by the previous TSE :smile: 

YOUR PROJECT HAS BEEN APPROVED
Go ahead and merge to master, kindly submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo)

**Pull request merged**